src/components/NewbieTasksList/NewbieTasksList.tsx

```diff
@@ -25,6 +26,9 @@ const NewbieTasksList: React.FC = () => {
     variables: { newbieId },
   });
 
+  const newbie = data ? data.newbie : undefined;
+  const { newbieProgress } = useTaskProgress(newbie);
```

Comment:
This will run even when we don't have newbie. Isn't a better way to run only this when we have in fact data?

Reply:
Fixed here (#90d27e2)[https://github.com/BuildIt-Poland/buddy-app/pull/61/commits/90d27e2ca5c70e562b45c061adbf45419e4df24f]